cg/cli/workflow/commands.py

```diff
@@ -262,3 +263,43 @@ def mutant_past_run_dirs(
     context.obj.meta_apis["analysis_api"] = MutantAnalysisAPI(context.obj)
 
     context.invoke(past_run_dirs, yes=yes, dry_run=dry_run, before_str=before_str)
+
+
+@click.group(invoke_without_command=True)
+@click.pass_context
+def fastq(context: click.Context):
```

Comment:
I wonder if the fastq, store and store-available should be placed in a workflow dir "fastq/base.py" instead since they are specific to the analysis_type "fastq" and you have to use @fastq.command to group these ones together.

Reply:
Sure, that makes sense